src/verifiers/issuerIdentity/index.ts

```diff
@@ -0,0 +1,27 @@
+import { issuerIdentityVerifierBuilder } from "./builder";
+import {
+  OpenAttestationDidIdentityProof,
+  OpenAttestationDnsDidIdentityProof,
+  OpenAttestationDnsTxtIdentityProof,
+} from "./verifiers";
+
+// Default set of issuer identity verifiers
+// Currently excludes OpenAttestationDidIdentityProof
+export const defaultIssuerIdentityProofVerifiers = [
+  OpenAttestationDnsDidIdentityProof,
+  OpenAttestationDnsTxtIdentityProof,
+];
+
+// Default ISSUER_IDENTITY verifier, built from defaultIssuerIdentityProofVerifiers
+export const OpenAttestationIssuerIdentityVerifier = issuerIdentityVerifierBuilder(defaultIssuerIdentityProofVerifiers);
+
+// Exporting all verifiers for dependant apps to compose
+export const verifiers = {
```

Comment:
Maybe we should be careful with this name because it's exported by the index file and then we have 3 sets of verifiers exported:

- verifiers
- defaultIssuerIdentityProofVerifiers
- openAttestationVerifiers

I fear that verifiers is hard to understand